src/crc.ts

```diff
@@ -0,0 +1,11 @@
+import { crc32 } from 'crc';
+
+class CRC {
+    public static crc32(data: string): number {
+        const buffer = Buffer.from(data, 'utf16le');
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-call
```

Comment:
Interesting that this is required even though you added a dependency on `@types/crc`. Do you know if eslint isn't working with `@types` dependencies or are the type definitions for `@types/crc` incomplete and the parameter types and return type is missing for the `crc32` function.

Reply:
Just figured out that it is not necessary. Good catch, thx